AppBrokerASP/DeviceManager.cs

```diff
@@ -291,5 +293,32 @@ public async Task GetZigbeeDevices(SocketIO socket)
                 }
             }
         }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposed)
+            {
+                if (disposing)
+                {
+                    // TODO: dispose managed state (managed objects)
+                    foreach (var device in Devices.Values)
+                    {
+                        device.Dispose();
```

Comment:
I would add an if(device is IDisposable disposable) disposable.Dispose(); and remove the IDisposable from the device, because most of the devices don't have anything that need disposing